### ISSUE TITLE
No error will be reported when In-Event is raised 

### ISSUE THREAD

**Issue by DefCon-CC** [Yakindu/statecharts#2718]
We observed the Problem in Yakindu SCT 3.4.1. that an In-Event can be raised. Which is from our point of view an error. Example:

![grafik](https://user-images.githubusercontent.com/25898240/54919076-2cb27d00-4f00-11e9-9833-3850ba3f0f03.png)

![grafik](https://user-images.githubusercontent.com/25898240/54919112-40f67a00-4f00-11e9-8d65-c65fd690ffde.png)

The code is generated without any further warning or error.

**Comment by BeckmaR** [comment]
Agree. At least a warning should be created, and we should deprecate this "feature" altogether.

**Comment by terfloth** [comment]
With release 4.x raising in events by the state machine itself is clearly defined using event buffering which is enabled by default. If it is disabled we should add warnings.
